src/resources/ha-style.html

```diff
@@ -5,53 +5,37 @@
   body {
     font-size: 14px;
 
-    /* for paper-toggle-button */
-    --paper-grey-50: #fafafa;
-    --paper-grey-200: #eeeeee;
-
-    --paper-item-icon-color: #44739e;
-    --paper-item-icon-active-color: #FDD835;
-
+    --primary-color: #03A9F4; /* header */
     --dark-primary-color: #0288D1;
-    --primary-color: #03A9F4;
     --light-primary-color: #B3E5FC;
-    --text-primary-color: #ffffff;
-    --accent-color: #FF9800;
-    --primary-background-color: var(--paper-grey-50);
-    --secondary-background-color: #E5E5E5;
+    --text-primary-color: #FFFFFF; /* header */
+    --primary-background-color: #fafafa;
     --primary-text-color: #212121;
+
+    --secondary-background-color: #E5E5E5; /* states, */
     --secondary-text-color: #727272;
+
+    --tertiary-background-color: #FFFFFF; /* badges, state cards, side menu */
```

Comment:
This is exactly why I don't like theming. It becomes messy really really fast.

I don't like tertiary background, I don't want to make it an "official" color. Let's just stick to individual variables for background badges, state cards and sidebar.

The original set of variables is based on what is used by the paper-* components by Google and what the material design themes prescribe https://www.materialpalette.com/blue/orange